src/Ttl.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Cache;
+
+final class Ttl
```

Comment:
Should we use this object for TTL parameters in other classes in this package?

Reply:
We can add new class usage without break BC, for example:

```php
public function __construct(\Psr\SimpleCache\CacheInterface $handler, DateInterval|int|null $defaultTtl = null)
// change to
public function __construct(\Psr\SimpleCache\CacheInterface $handler, Ttl|DateInterval|int|null $defaultTtl = null)
```

Also we can into classes cast all ttl values to `Ttl` object and use it inside already now.
 
By the way, method `fromInterval(DateInterval $interval)` is should be added.